### PR TITLE
 DDCE-773: Added Authorised official's phone number page

### PR DESCRIPTION
**Pull request by robert-webb-hmrc**
# DDCE-773

##New feature (delete as appropriate)

To add the authorised officials phone number to the journey.

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/forms/authorisedOfficials/AuthorisedOfficialsPhoneNumberFormProvider.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import forms.mappings.Mappings
+import javax.inject.Inject
+import models.AuthorisedOfficialsPhoneNumber
+import play.api.data.Form
+import play.api.data.Forms._
+
+class AuthorisedOfficialsPhoneNumberFormProvider @Inject() extends Mappings {
+
+  val validateTelephoneNumber = """^\+?(?:\s*\d){10,13}$"""
```
Comment (anburoach) [inline app/forms/authorisedOfficials/AuthorisedOfficialsPhoneNumberFormProvider.scala:27]: Plz change is variable to private like **private[authorisedOfficials] val validateTelephoneNumber**

Comment (robert-webb-hmrc) [reply app/forms/authorisedOfficials/AuthorisedOfficialsPhoneNumberFormProvider.scala]: Updated

File: test/models/AuthorisedOfficialsSpec.scala [hunk 2]
```diff
@@ -45,4 +44,29 @@ class AuthorisedOfficialsSpec extends WordSpec with MustMatchers with OptionValu
...
+      val authorisedOfficialsPhoneNumber = AuthorisedOfficialsPhoneNumber(
+        daytimePhone = "01632 960 001",
+        mobilePhone = Some("01632 960 001"))
+
+      authorisedOfficialsPhoneNumber.daytimePhone mustBe "01632 960 001"
+      authorisedOfficialsPhoneNumber.mobilePhone mustBe Some("01632 960 001")
+
+    }
+
+    "middleName is not defined" in {
```
Comment (anburoach) [inline test/models/AuthorisedOfficialsSpec.scala:61]: change the middleName to alternativePhoneNumber

Comment (robert-webb-hmrc) [reply test/models/AuthorisedOfficialsSpec.scala]: resolved

**Pull request merged**